SalesforceFoundation/k12-architecture-kit
# Renamed trial address layout and added to config

The address layout was not namespaced or referenced in the trial org config. This rectifies that issue.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
* Run regression org flow on a release shape org.
* Create a new address
* Verify that Mailing County is in the layout.
